build/mirror-repo.yml

```diff
@@ -0,0 +1,19 @@
+name: Mirror-to-client-repo
+
+on:
+  push:
+    branches:
+      - development
+
+jobs:
+  to_client_repo:
+    runs-on: ubuntu-latest
+    steps: # <-- must use actions/checkout before mirroring!
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: pixta-dev/repository-mirroring-action@v1
+        with:
+          target_repo_url: git@github.com:CLIENT_REPO/PROJECT.git # <-- need to change client's repo url
```

Comment:
```suggestion
          target_repo_url: ${{ secrets.CLIENT_REPO_GIT_URL }}
```

Reply:
We can use a secret for the URL instead of editing

Reply:
Nice